kodilla-stream/src/main/java/com/kodilla/stream/array/ArrayOperations.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.stream.array;
+
+import java.util.stream.IntStream;
+
+public interface ArrayOperations {
+    static double getAverage(int[] numbers) {
+        IntStream.rangeClosed(1, numbers.length)
+                .forEach(System.out::println);
+        IntStream.rangeClosed(1, numbers.length)
+                .average()
+                .getAsDouble();
+        return getAverage(numbers);
```

Comment:
Ta metoda nie powinna wolac sama siebie, tylko zwracac konkretna wartosc. getAsDouble zwraca optional, ktory trzeba rozpakowac i zwrocic
http://www.baeldung.com/java-optional
https://www.mkyong.com/java8/java-8-optional-in-depth/